kadi_apps/blueprints/find_attitude/find_attitude.py

```diff
@@ -130,9 +130,13 @@ def index():
     # Update some constraints to their default values if not set.
     for key, value in DEFAULT_CONSTRAINTS.items():
         form_val = request.form.get(key)
-        if form_val is not None and form_val.strip() != "":
-            value = float(form_val)
-            context[key] = value
+        if form_val is not None and form_val.strip() == "":
+            if key == 'maude_channel':
+                value = form_val
+                context[key] = value
+            else:
+                value = float(form_val)
+                context[key] = value
```

Comment:
The logic here can seemingly be simplified.
```
context.update(DEFAULT_CONSTRAINTS)
for key in DEFAULT_CONSTRAINTS:
    form_val = request.form.get(key)
    if form_val is not None and form_val.strip() != "":
        context[key] = ska_helpers.utils.convert_to_int_float_str(form_val)
```

Reply:
I like the idea of using convert_to_int_float_str, but using context.update()  and the logic on != "" means that empty fields get re-populated.

However, using context update and not bothering to do any special handling on != "" looks to basically work as expected.

One gotcha is that it looks like setting the attitude error to an empty field does not work as anticipated if the attitude field has a quaternion (throws an error).  However that looks like a pre-existing defect and not a regression.